app/services/data_requests/write_cloudwatch_logs.rb

```diff
@@ -18,23 +19,17 @@ def initialize(cloudwatch_results, output_dir)
     end
 
     def call
-      output_file.puts(HEADERS.join(','))
-      cloudwatch_results.each do |row|
-        write_row(row)
+      CSV.open(File.join(output_dir, 'logs.csv'), 'w') do |csv|
+        csv << HEADERS
+        cloudwatch_results.each do |row|
+          csv << build_row(row)
+        end
```

Comment:
Refactored existing code to pass around the built-in CSV generator object so we can pass around arrays, rather than having to worry about escaping individual print statements